pkg/resourcecollector/resourcecollector.go

```diff
@@ -522,6 +524,52 @@ func (r *ResourceCollector) GetResources(
 	return allObjects, pvcsWithOwnerReference, nil
 }
 
+// gatherResourceInChunks() collects all the resources present per ns or resource types.
+// It does it in batch of 500 unless specified by user via a config-map. This function
+// primarily helps in avoiding server timeout error when the number of resource or size
+// of it large.
+func gatherResourceInChunks(dynamicClient dynamic.ResourceInterface, opts Options, selectors string) (*unstructured.UnstructuredList, error) {
+	var err error
+	fn := "gatherResourceInChunks"
+	// Read configMap for any user provided resource count limit to be provided to List call.
+	resourceCountLimit := opts.ResourceCountLimit
+	resourceCountLimitString, err := k8sutils.GetConfigValue(k8sutils.StorkControllerConfigMapName, metav1.NamespaceSystem, k8sutils.ResourceCountLimitKeyName)
+	if err != nil {
+		logrus.Infof("error in reading %v cm for the key %v, switching to default value passed to GetResource API",
```

Comment:
Please include the err in the debug statement. It is better to change Infof to Warnf or Debugf

Reply:
Done.